adapter_ninafw.go

```diff
@@ -11,6 +11,29 @@ import (
 
 const maxConnections = 1
 
+// NINAConfig encapsulates the hardware options for the NINA firmware
+type NINAConfig struct {
```

Comment:
I suggest name this `HCIConfig` so it can be more easily used for non-NINA HCI controllers.